app/lib/tool/test_profile/import_source.dart

```diff
@@ -146,7 +146,10 @@ class _AutoGeneratedImportSource implements ImportSource {
           'path': '${screenshot.path}',
           'description': '${screenshot.description}'
         }
-      ]
+      ],
+      'funding': [
```

Comment:
Can we make optional, so that not all test packages have this? Maybe just the ones named `oxygen` ?

Reply:
I'd like to semi-randomize it (same with screenshots and other items), but probably in a separate PR, because it would impact a lots of tests.